src/js/AppContainer.jsx

```diff
@@ -41,6 +41,7 @@ var AppContainer = React.createClass({
       <div>
         {selectComponent}
         <UserFiHeading institution={{name: params.encodedInstitutionName}} year="2017" user={this.state.user.name}/>
+        /*clones any children added by the router in order to pass them props*/
```

Comment:
Your comment needs to be `{/*clones any children added by the router in order to pass them props*/}` (wrapped in `{}`) so that it doesn't show up in the UI.